code/game/objects/items/storage/medkit.dm

```diff
@@ -803,11 +803,12 @@
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
 	w_class = WEIGHT_CLASS_SMALL
+	var/max_slots = 8
```

Comment:
This isn't a clean fix. After this var is read it goes unused forever.

You should instead create a subtype for test tube storage inside `code/datums/storage/subtypes` folder something like this

```
/datum/storage/test_tube_rack
	max_slots = 8
	screen_max_columns = 4
	screen_max_rows = 2
	allow_quick_gather = TRUE

/datum/storage/test_tube_rack/Initialize()
	. = ..()
	set_holdable(/obj/item/reagent_containers/cup/tube)
```

And then assign this storage type to the test tube rack like such
```
/obj/item/storage/test_tube_rack
	storage_type = /datum/storage/test_tube_rack
```

Reply:
ouagh yeah I forgot we had that, thanks for reminding

Reply:
new rather than initialize, and also removed the attack_self empty storage bit given allow_quick_empty on the storage handles this already, so nice